src/common/utils/utils.ts

```diff
@@ -286,25 +292,46 @@ export function getNextPageParam(lastPage?: GenericResponseType<any>) {
   return sum;
 }
 
-export const numberToString = (value: number) => {
```

Comment:
This was incorrectly handling numbers whose decimals were greater than 3, and not ideally handling trailing zeroes 
![image](https://github.com/user-attachments/assets/facc871a-0d8a-4216-beef-67d84d4cdab9)